app/src/main/java/com/mapbox/vision/examples/activity/ar/ArMapActivity.kt

```diff
@@ -149,7 +165,7 @@ class ArMapActivity : AppCompatActivity(), MapboxMap.OnMapClickListener,
 
                     // Draw the route on the map
                     if (navigationMapRoute != null) {
```

Comment:
```
navigationRoute?.let{ navRoute -> 
  navRoute.updateRouteVisibilityTo(false)
} ?: let {
  navigationRoute = NavigationMapRoute(null, mapView, mapboxMap, R.style.NavigationMapRoute)
}
```

Reply:
don't use unsafe `!!`

Reply:
@RingerJK simple if-else has better readability then 2 let and elvis operator.
also after if-else _navigationRoute_ is not null, so !! is safe

Reply:
`!!` is bad practice. just try to avoid it